From faef85253e3770b59cac6ec0a7326ce64f66e383 Mon Sep 17 00:00:00 2001
From: Yassin <yassin@Nixos>
Date: Mon, 24 Aug 2026 16:03:09 +0200
Subject: [PATCH 3/6] mmc: mtk-sd: add MT6765 compatibility

MT6765 uses the same controller capabilities as MT6779, including the
12-bit divider, asynchronous FIFO and enhanced receive path.
---
 drivers/mmc/host/mtk-sd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/mtk-sd.c b/drivers/mmc/host/mtk-sd.c
index b2680cc05..aba4facc0 100644
--- a/drivers/mmc/host/mtk-sd.c
+++ b/drivers/mmc/host/mtk-sd.c
@@ -715,6 +715,7 @@ static const struct mtk_mmc_compatible mt8196_compat = {
 static const struct of_device_id msdc_of_ids[] = {
 	{ .compatible = "mediatek,mt2701-mmc", .data = &mt2701_compat},
 	{ .compatible = "mediatek,mt2712-mmc", .data = &mt2712_compat},
+	{ .compatible = "mediatek,mt6765-mmc", .data = &mt6779_compat},
 	{ .compatible = "mediatek,mt6779-mmc", .data = &mt6779_compat},
 	{ .compatible = "mediatek,mt6795-mmc", .data = &mt6795_compat},
 	{ .compatible = "mediatek,mt7620-mmc", .data = &mt7620_compat},
